DigitalLearningSolutions.Data.Tests/Services/SectionServiceTests.cs

```diff
@@ -98,9 +98,67 @@ public void GetSectionAndTutorialsBySectionIdForCustomisation_returns_null_if_se
             using (new AssertionScope())
             {
                 result.Should().BeNull();
-                A.CallTo(() => tutorialContentDataService.GetTutorialsBySectionId(A<int>._, 1))
+                A.CallTo(() => tutorialContentDataService.GetTutorialsBySectionIdAndCustomisationId(A<int>._, 1))
                     .MustNotHaveHappened();
             }
         }
+
+        [Test]
+        public void GetSectionsForApplication_calls_data_service()

```

Comment:
Similar to the other test, the name of this test could be changed so that it better describes the full scope of the test, e.g.:
`GetSectionsForApplication_calls_data_service_and_returns_expected_sections()`

Reply:
done